tests/flow/async_iteration/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -134,9 +134,11 @@ declare var gen: AsyncGenerator<void, string, void>;
 
 // You can pass whatever you like to return, it doesn\'t need to be related to
 // the AsyncGenerator\'s return type
-gen.return(0).then(result => {
-  (result.value: void); // error: string | number ~> void
-});
+gen
+  .return(0)
+  .then(result => {
+    (result.value: void); // error: string | number ~> void
+  });
```

Comment:
I can special case if only the last call has a function as only argument. Right now if you've got functions in between, it's not going to indent them properly.